docs/logs.md

```diff
@@ -0,0 +1,79 @@
+<!--
+---
+linkTitle: "Logs"
+weight: 4
+---
+-->
+
+# Tekton Dashboard log viewer
+
+This guide describes the features and functionality of the log viewer provided by the Tekton Dashboard on the `TaskRun` and `PipelineRun` details pages.
+
+## Basic functionality
+
+The Tekton Dashboard log viewer supports ANSI colour codes and text styles, and will automatically detect URLs in log content and render them as clickable links opening in a new window.
+
+## Toolbar
+
+The toolbar diplayed in the log viewer includes a number of additional features, including:
+
+- maximize: increase the area available to the log viewer by hiding the task list and run header. This allows the user to eliminate distractions from other parts of the app and focus on the log content.
+- open in new window: open the raw logs in a separate browser window. This provides an unmodified and unprocessed view of the logs, without any of the added features provided by the log viewer.
+- download: download the raw logs as a text file.
+- user preferences: these are persisted to browser local storage and applied to all logs in the app. See the following sections for more details.
+
+### Timestamps
+
+The Dashboard will always request logs with timestamps from the Kubernetes pod logs API, and show / hide the timestamps in the log viewer based on the user's preference. This can be toggled from the settings menu in the toolbar at the top of the log viewer.
+
+The timestamps are localised based on users' browser settings, with the raw timestamp value received from the API provided as a tooltip on hover.
```

Comment:
Was this intentional usage of plural `users'` ownership apostrophe? I think singular reads a bit better?
```suggestion
The timestamps are localised based on the user's browser settings, with the raw timestamp value received from the API provided as a tooltip on hover.
```

Reply:
Yeah either works, and I notice I used the singular in the previous paragraph so will update here too. Thanks 👍